src/scss/base.scss

```diff
@@ -93,6 +93,49 @@ body {
   margin: 0 0 15px 0;
 }
 
+.footer-container {
+  background: $orange;
+  height: 100%;
```

Comment:
you don't need to set a specific height for this, just let the content adapt and use padding to separate the element from the borders.